Composer/packages/client/src/utils/convertUtils/parsePathToFocused.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { parsePathToSelected } from './parsePathToSelected';
+
+export function parsePathToFocused(path: string): string {
+  //path is like main.trigers[0].actions[0]
+
+  const trigger = parsePathToSelected(path);
+
+  const list = path.split('.');
+
+  const matchActions = list.filter(x => x.startsWith('actions') || x.startsWith('elseActions'));
```

Comment:
`actions` and `elseActions` should come off of the schema and not hardcoded here